usaspending_api/common/helpers/generic_helper.py

```diff
@@ -122,14 +123,28 @@ def dates_are_month_bookends(start, end):
     return False
 
 
-def generate_all_fiscal_years_in_range(start, end):
-    """ For a given date-range, provide the inclusive fiscal years """
-    fiscal_years = []
+def hash_date_range(date_range_dict):
```

Comment:
Confused by the function name

Reply:
Changed name to `generate_date_range_hash` with a comment to explain: "For a given date range dictionary, make a string that can be used as a hash and sort a list of date ranges"